drivers/homeless_summary_report/app/views/homeless_summary_report/warehouse_reports/reports/_form.haml

```diff
@@ -11,7 +12,7 @@
         = f.input :end, as: :date_picker, label: 'End Date', required: true
       .col-sm-6
         = f.input :coc_codes, collection: @filter.coc_code_options_for_select(user: current_user), as: :select_two, input_html: {multiple: true, placeholder: 'Choose CoCs', data: {'filter-projects-target' => 'cocCodes', 'action': 'change->filter-projects#update'}}, label: 'CoC Codes'
-    = render 'hud_reports/project_filter', f: f
+    = render 'hud_reports/project_filter', f: f, project_types_collection: @report.project_type_options_for_select, projects_collection: @report.project_options_for_select(current_user), alert: alert
```

Comment:
rather than pass in the project list, can we just fetch it remotely?  There's an example in the style guide.

Reply:
Done with https://github.com/greenriver/hmis-warehouse/pull/2199/commits/40c40b0cd3e487f0830d1f3f9186319f5f4d3959

There were a few changes needed:
* Update the endpoint to accept `funder_codes`
* Update the shared `app/views/hud_reports/_project_filter.haml` to accept an object `api_projects_parameters`that it will pass along to the `api_projects_path` endpoint. This will be flexible for the future if we want to filter project list by organization, coc, data source, etc. 
* Add a hidden `funder_ids` field to the PATH form, so that the "Included Projects" list limits projects by funder too

Reply:
Cleaned up those templates with https://github.com/greenriver/hmis-warehouse/pull/2199/commits/116ccc3d52ce74efbf4143e5125865930977cf78